lib/beautiful/xresources.lua

```diff
@@ -83,8 +83,16 @@ function xresources.get_dpi(s)
         if awesome and awesome.xrdb_get_value then
             xresources.dpi = tonumber(awesome.xrdb_get_value("", "Xft.dpi"))
```

Comment:
What does "link in comment" in the commit message refer to? What kind of comment can a commit message have?